docs/tutorial/README.md

```diff
@@ -0,0 +1,119 @@
+# Dapr Shared Getting Started Tutorial
+
+This tutorial follows up on the Dapr Kubernetes tutorial that can be located here: 
+
+https://github.com/dapr/quickstarts/tree/master/tutorials/hello-kubernetes
+
+Instead of deploying Dapr as a sidecar we are going to use Dapr Shared instances. 
+
+
+## Prerequisites and Installation
+
+Before proceeding, please make sure that you have the necessary tools installed on your system. We will create a local KinD cluster to install Dapr, some applications, and an instance of Dapr Shared.
+
+To get started, make sure you have the following CLIs installed:
+
+- [Docker](https://www.docker.com/)
+
+- [KinD (Kubernetes in Docker)](https://kind.sigs.k8s.io/docs/user/quick-start/)
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+- [Helm](https://helm.sh/docs/intro/install/)
+
+
+## Creating a local Kubernetes cluster with KinD: 
+
+Here, you will create a simple Kubernetes cluster with KinD defaults running the following command:
+
+```bash
+kind create cluster
+```
+
+Make sure to install the Dapr Control Plane into the cluster with:
+
+```
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+helm upgrade --install dapr dapr/dapr \
+--version=1.13.2 \
+--namespace dapr-system \
+--create-namespace \
+--wait
+```
+
+## Running the Hello example
+
+We will be using the Dapr Statestore API and for that we will install a Redis instance into our cluster using Helm: 
+
+```shell
+helm install redis oci://registry-1.docker.io/bitnamicharts/redis --version 17.11.3 --set "architecture=standalone" --set "master.persistence.size=1Gi"
+```
+
+Once Redis is installed we can deploy our application workloads, including the Statestore component by running: 
```

Comment:
```suggestion
Once Redis is installed you can deploy our application workloads, including a statestore component by running: 
```

Reply:
By the way the easiest way to get this set up for a cluster with redis is

`dapr init -k --dev`

Reply:
@msfussell I've created an issue in the Hello Kubernetes tutorial.. because that's not how Kubernetes users do that. For people coming from that space using `dapr init -k` is super confusing.